packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -61,22 +122,34 @@ abstract class InstanceEngineBase implements IInstanceEngine {
   public readonly singleUserRotationApplication: secretsmanager.SecretRotationApplication;
   public readonly multiUserRotationApplication: secretsmanager.SecretRotationApplication;
 
+  private readonly features?: InstanceEngineFeatures;
+
   constructor(props: InstanceEngineBaseProps) {
     this.engineType = props.engineType;
+    this.features = props.features;
     this.singleUserRotationApplication = props.singleUserRotationApplication;
     this.multiUserRotationApplication = props.multiUserRotationApplication;
     this.engineVersion = props.version;
     this.parameterGroupFamily = props.parameterGroupFamily ??
       (this.engineVersion ? `${this.engineType}${this.engineVersion.majorVersion}` : undefined);
   }
 
+  public get description() {
+    return this.engineType + (this.engineVersion ? `-${this.engineVersion?.fullVersion}` : '');
```

Comment:
I know this might be surprising, but [`fullVersion` is actually optional!](https://github.com/aws/aws-cdk/blob/a5e77e47d3bc7a326be56eef527db6f988b8ac9f/packages/%40aws-cdk/aws-rds/lib/engine-version.ts#L6-L14) (this is used in engines that have a single major version, like [Aurora MySQL](https://github.com/aws/aws-cdk/blob/a5e77e47d3bc7a326be56eef527db6f988b8ac9f/packages/%40aws-cdk/aws-rds/lib/cluster-engine.ts#L372-L372))

Reply:
```ts
export function engineDescription(engine: IInstanceEngine) {
  return engine.engineType + (engine.engineVersion?.fullVersion ? `-${engine.engineVersion.fullVersion}` : '');
}
```